client/packages/host/src/components/Login/LoginError.tsx

```diff
@@ -61,9 +66,18 @@ export const LoginError = ({ error, details, hint }: LoginErrorProps) => {
             >
               {t('error.more-info')}
               {expand ? (
-                <ChevronDownIcon />
+                <ChevronDownIcon
+                  sx={{
+                    ...chevronCommonStyles,
+                  }}
+                />
               ) : (
-                <ChevronDownIcon sx={{ transform: 'rotate(-90deg)' }} />
+                <ChevronDownIcon
+                  sx={{
+                    transform: 'rotate(-90deg)',
+                    ...chevronCommonStyles,
+                  }}
+                />
```

Comment:
From https://github.com/msupply-foundation/open-msupply/pull/5030#issuecomment-2400803665

Reply:
![Screenshot 2024-10-11 at 1 07 04 PM](https://github.com/user-attachments/assets/b6ee2324-1922-4798-870a-569a5e280145)

I'm gonna nit bc I know you get it... can it be centre aligned 😭

Reply:
Yah it's centred align on mine 👀

Reply:
Weird, all good then 😁